README.md

```diff
@@ -37,13 +38,22 @@ CASA participants are structured into three groups:
 
 ### **CASA Meetings**
 
-All CASA stakeholders are invited to join a meeting every six weeks (attendance optional), for the purpose of:
+All CASA stakeholders are invited to join a meeting every two weeks, for the purpose of:
```

Comment:
```suggestion
All CASA stakeholders are invited to join a meeting every two weeks, for the purpose of:
```
should we really do 2 weeks - I would rather go monthly to be honest - 2 weeks sounds very ambitious - no "over my dead body" - just my feeling.

Reply:
i don't have strong feelings, but if the group lands on monthly let's call it "every 4 weeks" operationally instead of "first X of each month" because in my experience it's a lot simpler to handle holidays and such.